src/reactionsystem_conversions.jl

```diff
@@ -43,22 +48,33 @@ function oderatelaw(rx; combinatoric_ratelaw = true)
 end
 
 # Function returning `true` for species which shouldn't change from the reactions, 
-# including non-species variables.
-drop_dynamics(s) = isconstant(s) || isbc(s) || (!isspecies(s))
-
-function assemble_oderhs(rs, ispcs; combinatoric_ratelaws = true, remove_conserved = false)
+# including non-species variables
+drop_dynamics(spec) = isconstant(spec) || isbc(spec) || (!isspecies(spec))
```

Comment:
I tried to reduce the number of expressions used to describe a species. Here `spec` was used in all places where `drop_dynamics` was called.